api/operator/v1/vmextra_types.go

```diff
@@ -0,0 +1,58 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type AssetKind string
+
+const (
+	TLSAssetKind     AssetKind = "tls"
+	ConfigAssetKind  AssetKind = "config"
+	DefaultAssetKind AssetKind = "default"
+)
+
+// +kubebuilder:object:generate=false
+// +kubebuilder:pruning:PreserveUnknownFields
+// +kubebuilder:validation:Schemaless
+type Args map[string]any
```

Comment:
This struct is not used by `api` package. It also do not have proper comment. Consider to move it to the `build` or any other suitable place.

Reply:
removed it, was used for vmanomaly model CRs